crates/router/src/connector/payme/transformers.rs

```diff
@@ -507,6 +586,88 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for PayRequest {
         }
     }
 }
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct PaymePayloadData {
+    meta_data: String,
+}
+
+impl TryFrom<&types::PaymentsCompleteAuthorizeRouterData> for Pay3dsRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::PaymentsCompleteAuthorizeRouterData) -> Result<Self, Self::Error> {
+        match item.request.payment_method_data.clone() {
+            Some(api::PaymentMethodData::Card(_)) => {
+                let buyer_email = match item.request.email.clone() {
+                    Some(email) => email,
+                    None => Err(errors::ConnectorError::MissingRequiredField {
+                        field_name: "email",
+                    })?,
+                };
+                let buyer_name = item.get_billing_address()?.get_full_name()?;
+
+                let payload_data = item
+                    .request
+                    .redirect_response
+                    .to_owned()
+                    .and_then(|response| response.payload.map(|param_string| param_string.expose()))
+                    .ok_or(errors::ConnectorError::MissingConnectorRedirectionPayload {
+                        field_name: "meta_data",
+                    })?;
+
+                let jwt_data: PaymePayloadData = serde_json::from_value(payload_data)
+                    .into_report()
+                    .change_context(errors::ConnectorError::MissingConnectorRedirectionPayload {
+                        field_name: "meta_data_jwt",
+                    })?;
+
+                let payme_sale_id = item
+                    .request
+                    .connector_transaction_id
+                    .clone()
+                    .ok_or(errors::ConnectorError::MissingConnectorTransactionID)?;
+                let buyer_key = match item.payment_method_token.clone() {
+                    Some(key) => key,
+                    None => Err(errors::ConnectorError::MissingRequiredField {
+                        field_name: "buyer_key",
+                    })?,
+                };
+                Ok(Self {
+                    buyer_email,
+                    buyer_key,
+                    buyer_name,
+                    payme_sale_id,
+                    meta_data_jwt: jwt_data.meta_data,
+                })
+            }
+            _ => Err(
```

Comment:
please do not use default implementation. Instead map rest other payment methods to Not Implemented